changes/ocd-4389.sql

```diff
@@ -0,0 +1,120 @@
+SELECT add_version_to_cqm('CMS147', 'v13', 'v12');
+SELECT add_version_to_cqm('CMS149', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS153', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS154', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS155', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS156', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS157', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS159', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS161', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS165', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS177', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS249', 'v6', 'v5');
+SELECT add_version_to_cqm('CMS347', 'v7', 'v6');
+SELECT add_version_to_cqm('CMS349', 'v6', 'v5');
+SELECT add_version_to_cqm('CMS645', 'v7', 'v6');
+SELECT add_version_to_cqm('CMS646', 'v4', 'v3');
+SELECT add_version_to_cqm('CMS771', 'v5', 'v4');
+SELECT add_version_to_cqm('CMS71', 'v13', 'v12');
+SELECT add_version_to_cqm('CMS72', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS104', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS108', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS190', 'v12', 'v11');
+SELECT add_version_to_cqm('CMS506', 'v6', 'v5');
+SELECT add_version_to_cqm('CMS529', 'v4', 'v3');
+SELECT add_version_to_cqm('CMS844', 'v4', 'v3');
+SELECT add_version_to_cqm('CMS951', 'v2', 'v1');
+SELECT add_version_to_cqm('CMS334', 'v5', 'v4');
+SELECT add_version_to_cqm('CMS816', 'v3', 'v2');
+SELECT add_version_to_cqm('CMS871', 'v3', 'v2');
+SELECT add_version_to_cqm('CMS1028', 'v2', 'v1');
+SELECT add_version_to_cqm('CMS996', 'v4', 'v3');
+
+drop function add_version_to_cqm;
+
+
+INSERT INTO openchpl.cqm_criterion (cms_id, title, description, cqm_domain, nqf_number, last_modified_user, cqm_version_id, cqm_criterion_type_id, retired)
+  SELECT 'CMS1188', 
+	'Sexually Transmitted Infection (STI) Testing for People with HIV', 
+	'Percentage of patients 13 years of age and older with a diagnosis of HIV who had tests for syphilis, gonorrhea, and chlamydia performed within the measurement period', 
+	'Health Resources & Services Administration ', 
+	'N/A', 
+	-1, 
+	(SELECT cqm_version_id FROM openchpl.cqm_version WHERE version = 'v1'), 
+	(select cqm_criterion_type_id from openchpl.cqm_criterion_type where name = 'Ambulatory'), 
+	false
+  WHERE NOT EXISTS (select * from openchpl.cqm_criterion where cms_ID = 'CMS1188');
+
+INSERT INTO openchpl.cqm_criterion (cms_id, title, description, cqm_domain, nqf_number, last_modified_user, cqm_version_id, cqm_criterion_type_id, retired)
+  SELECT 'CMS819', 
+	'Hospital Harm - Opioid-Related Adverse Events', 
+	'This measure assesses the number of inpatient hospitalizations for patients age 18 and older who have been administered an opioid medication and are subsequently administered an opioid antagonist within 12 hours, an indication of an opioid-related adverse event', 
+	null, 
+	'3501e', 
+	-1, 
+	(SELECT cqm_version_id FROM openchpl.cqm_version WHERE version = 'v2'), 
```

Comment:
Seems weird that we're creating a CQM with a version 2, but no version 1. I'm assuming this isn't a typo, but is just weird data, but wanted to mention it.

Same thing with CMS986